presto-benchto-benchmarks/generate_schemas/generate-tpcds.py

```diff
@@ -2,9 +2,13 @@
 
 schemas = [
     # (new_schema, source_schema)
-    ('tpcds_10gb_orc', 'tpcds.sf10'),
-    ('tpcds_100gb_orc', 'tpcds.sf100'),
-    ('tpcds_1tb_orc', 'tpcds.sf1000'),
+    ('tpcds_sf10_orc', 'tpcds.sf10'),
+    ('tpcds_sf30_orc', 'tpcds.sf30'),
+    ('tpcds_sf100_orc', 'tpcds.sf100'),
+    ('tpcds_sf300_orc', 'tpcds.sf300'),
+    ('tpcds_sf1000_orc', 'tpcds.sf1000'),
+    ('tpcds_sf3000_orc', 'tpcds.sf3000'),
+    ('tpcds_sf10000_orc', 'tpcds.sf10000'),
```

Comment:
cc @sopel39 @ilfrin this makes it consistent with
https://github.com/prestosql/presto/blob/e6731d0d1aa12804eda6cf482af01de3981a6e31/presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpcds.yaml#L8-L14
the schema names were changed there in b238a3de47f811ca881ea4bf7db77d99399ae1ff

@willshen could you please change commit message to something like
```
Update schemas in generate-tpcds.py

This makes the script generate the schemas as required by
presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpcds.yaml
```